src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceCallbackHandler.java

```diff
@@ -62,6 +71,44 @@
         );
     }
 
+    private CallbackResponse populateRespondent1ClaimResponseDocumentCopy(CallbackParams callbackParams) {
+        var caseData = callbackParams.getCaseData();
+
+        CaseData.CaseDataBuilder updatedData =
+            caseData.toBuilder()
+                .respondent1ClaimResponseDocumentCopy(caseData.getRespondent1ClaimResponseDocument());
+
+        MultiPartyScenario multiPartyScenario = getMultiPartyScenario(caseData);
+
+        if (featureToggleService.isMultipartyEnabled()) {
+            switch (multiPartyScenario) {
+                case TWO_V_ONE:
+                    updatedData
+                        .claimantResponseScenarioFlag(ClaimantResponseScenarioFlag.TWO_V_ONE)
+                        .build();
+                    break;
+                case ONE_V_TWO_ONE_LEGAL_REP:
+                    updatedData
+                        .claimantResponseScenarioFlag(ClaimantResponseScenarioFlag.ONE_V_TWO_SAME_SOLICITOR)
+                        .build();
```

Comment:
instead of introducing this flag in the last leg of journey, can we introduce a multipartyScenario flag in createclaim and use it all throughout the journey later. We can remove this getMultiPartyScenario(caseData) call then from all places where used

Reply:
I was thinking to use MultiPartyScenario enum instead of the new enum. but introducing the flag in the create journey sounds out of scope?

Reply:
For the scope of this ticket, maybe  introducing the flag in the create journey is out of scope. But we should definitely try to reuse the MultPartyScenario enum instead of creating a new similar enum ClaimantResponseScenarioFlag